### ISSUE TITLE
File with only non imported films

### ISSUE THREAD

**Issue by MrModest** [StegSchreck/RatS#63]
We have only full film list json now. 
May be should make copy of this file and deleting from it film after marked. 
With this file we could continue where stopped rather than starting from zero.

**Comment by StegSchreck** [comment]
@MrModest Good idea. I will take this feature into consideration for the future improvement.
